FND/ProductsAndServices/PaymentsAndSchedules.rdf

```diff
@@ -115,34 +116,16 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-pas-psch;Payment">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;AdHocScheduleEntry"/>
```

Comment:
I did that because of feedback in the LOAN working group, which wanted the change.  There was significant discussion in JIRA about it in fact.  I'll change it back for now until we see how this plays out, but it was a specific request by the LOAN working group who modeled their own Payment in LoanCore because they didn't like this definition.  I think it could go either way and the one could add either ad hoc or regular schedule entry in cases where that is warranted, so I am ok with going back on this one.

Reply:
Maybe we need a subclass of Payment? What was the Jira issue that had the discussion?